src/sentry/replays/endpoints/project_replay_breadcrumbs.py

```diff
@@ -0,0 +1,246 @@
+        "is_dead",
+        "is_rage",
+        "selector",
+        "alt",
+        "aria_label",
+        "component_name",
+        "class",
+        "id",
+        "role",
+        "title",
+        "testid",
+    ],
+    "navigation": ["from", "to"],
+    "resource.xhr": [
+        "url",
+        "method",
+        "duration",
+        "statusCode",
+        "request_size",
+        "response_size",
+    ],
+    "resource.fetch": [
+        "url",
+        "method",
+        "duration",
+        "statusCode",
+        "request_size",
+        "response_size",
+    ],
+    "resource.script": [
+        "url",
+        "duration",
+        "size",
+        "statusCode",
+        "decodedBodySize",
+        "encodedBodySize",
+    ],
+    "resource.img": [
+        "url",
+        "duration",
+        "size",
+        "statusCode",
+        "decodedBodySize",
+        "encodedBodySize",
+    ],
+    "web-vital.cls": ["duration", "rating", "size", "value"],
+    "web-vital.fcp": ["duration", "rating", "size", "value"],
+    "web-vital.lcp": ["duration", "rating", "size", "value"],
+    "replay.hydrate-error": ["url"],
+    "replay.mutations": ["count"],
+    "sdk.options": [
+        "shouldRecordCanvas",
+        "useCompressionOption",
+        "blockAllMedia",
+        "maskAllText",
+        "maskAllInputs",
+        "useCompression",
+        "networkDetailHasUrls",
+        "networkCaptureBodies",
+        "networkRequestHasHeaders",
+        "networkResponseHasHeaders",
+        "sessionSampleRate",
+        "errorSampleRate",
+    ],
+    "memory": ["jsHeapSizeLimit", "totalJSHeapSize", "usedJSHeapSize", "endTimestamp", "duration"],
+}
+
+
+@region_silo_endpoint
+@extend_schema(tags=["Replays"])
+class ProjectReplayBreadcrumbsEndpoint(ProjectEndpoint):
+    owner = ApiOwner.REPLAY
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+    }
+
+    def get(self, request: Request, project: Project, replay_id: str) -> Response:
+        if not features.has(
+            "organizations:session-replay", project.organization, actor=request.user
+        ):
+            return Response(status=404)
+
+        filter_params = self.get_filter_params(request, project)
+        end, start = filter_params["end"], filter_params["start"]
+
+        def data_fn(offset: int, limit: int):
+            request_meta: RequestMeta = {
+                "cogs_category": "replay",
+                "debug": False,
+                "end_datetime": end,
+                "organization_id": project.organization_id,
+                "project_ids": [project.id],
+                "referrer": "replays.details.breadcrumbs.list",
+                "request_id": uuid.uuid4().hex,
+                "start_datetime": start,
+                "trace_item_type": "replay",
+            }
+
+            snuba_query = Query(
+                match=Entity("trace_items"),
+                select=[Column(k) for k in QUERY_SETTINGS["attribute_types"].keys()],
```

Comment:
is there a 'select all' equivalent? that might be handy to transition from what we have now into something that leverages EAP fully

Reply:
This is select all.  So you get all the breadcrumb attributes.  I need to add more test coverage to make this more clear (and test for regression!).